README.md

```diff
@@ -1581,35 +1613,6 @@ Selects the rightmost point of each series.
 
 Selects the highest point of each series.
 
-### Plot.select(*selector*, *options*)
-
-Selects the points of each series selected by the *selector*, which can be specified either as a function which receives as input the index of the series, or as a key: value object with exactly one key representing a channel, and the value being a function which receives as inputs the index of the series and the channel or the short-hand "min" and "max" which respectively select the least and greatest points for the specified channel.
-
-For example, to select the point within each series that is the closest to the median of the *y* channel:
-
-```js
-  Plot.select({
-    y: (I, V) => {
-      const median = d3.median(I, i => V[i]);
-      const i = d3.least(I, (i) => Math.abs(V[i] - median));
-      return [i];
-    }, {
-      x: "year",
-      y: "revenue",
-      fill: "format"
-    })
```

Comment:
I fixed the missing brace here, and removed the extra indentation (which was repeated below).